contracts/interchain-token-service/src/contract/execute/mod.rs

```diff
@@ -408,6 +424,52 @@ fn update_chain(storage: &mut dyn Storage, config: msg::ChainConfig) -> Result<(
     Ok(())
 }
 
+pub fn register_p2p_token_instance(
+    deps: DepsMut,
```

Comment:
```suggestion
    storage: &mut dyn Storage,
```
for consistency

Reply:
the pub functions in this file actually take `DepsMut` and only the private functions accept `Storage`, so it's actually consistent as is